app/assets/stylesheets/components/_intervention.scss

```diff
@@ -0,0 +1,3 @@
+.gem-c-intervention {
```

Comment:
We can remove this stylesheet by using the spacing classes from the design system, see further info below.